spyder/plugins/outlineexplorer/api.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""Outline explorer API."""
+
+
+class OutlineExplorerServer(object):
```

Comment:
@goanpeca this is the idea of the API of the outlineexplorer for https://github.com/spyder-ide/spyder-notebook/issues/91
What do you think?

Although this is not yet ready, there is some things missing in the internal behavior of the OutlineExplorerWidget